### ISSUE TITLE
This project very similar with standalone-migrations

### ISSUE THREAD

**Issue by zw963** [rosenfeld/active_record_migrations#24]
Hi, I see this project after i use [this](https://github.com/thuss/standalone-migrations) for a while with roda.

Any relation about those two project? 

Thanks

**Comment by rosenfeld** [comment]
For my new Roda apps I'm no longer using this project though but sequel_tools instead.

https://github.com/rosenfeld/sequel_tools

**Comment by zw963** [comment]
@rosenfeld , hi, I use `standalone-migrations` because I want avoid some issue with sequel builtin migration,
And, ActiveRecord migration is feature riched, and more accuracy when generate `db/schema.rb` compare to sequel.

anyway, thanks for advice.

**Comment by rosenfeld** [comment]
Yes, if supporting a scheme definition in Ruby is important to you I'd advise you to use AR. I've never used schema.rb in my projects, having opted for using the structure.sql dump instead as the Ruby schema dump has always been limited for my use cases even with ActiveRecord.
